lib/src/settings.rs

```diff
@@ -242,12 +242,24 @@ impl UserSettings {
             .unwrap_or_else(|_| "curved".to_string())
     }
 
-    pub fn default_node_symbol(&self) -> String {
-        self.node_symbol_for_key("ui.graph.default_node", "◉", "o")
+    pub fn commit_node_template(&self) -> String {
+        self.node_template_for_key(
+            "ui.graph.commit-node-template",
+            r#"if(current_working_copy, "@", if(immutable, "◉", "◉"))"#,
+            r#"if(current_working_copy, "@", if(immutable, "o", "o"))"#,
+        )
```

Comment:
nit: it would be nice if we can move them to `config/templates.toml`, but there are `"ascii"` stuff.

Do we still need them to be the graph-style dependent by default?

Reply:
Having sensible defaults that follow from picking the graph style seems important, though I agree having to put defaults here is suboptimal.

I guess one option could be to split the template in 2 to have a dedicated ascii variant in the config. Or there could be an `ascii` property available in the template (though then we are back to extending template languages again, which is a bit iffy atm).

Reply:
Ok, let's leave things as is for clean-up later. When we have the ability to extend languages we can add an elided property along with an ascii property, and then the default can be moved to the toml.